protocol/encryption/protocol.go

```diff
@@ -284,6 +291,34 @@ func (p *Protocol) BuildHashRatchetMessage(groupID []byte, payload []byte) (*Pro
 	return spec, nil
 }
 
+func (p *Protocol) GetKeyExMessageSpecs(communityID []byte, identity *ecdsa.PrivateKey, recipients []*ecdsa.PublicKey, forceRenew bool) ([]*ProtocolMessageSpec, error) {
+	var communityKeyID uint32
+	var err error
+	if !forceRenew {
```

Comment:
Rekey maybe instead of renew?

Reply:
Thanks, totally agree, `Encrypted` in db is useless. Changed that, and also removed the `Encrypted` field from communities db table and added it to `CommunityDescription`.